backend/azure/dtmb/internal/utils/connectworker.go

```diff
@@ -0,0 +1,248 @@
+package utils
+
+import (
+	"context"
+	"crypto/rand"
+	"encoding/base64"
+	"errors"
+	"fmt"
+	"io"
+	"log"
+	"strings"
+	"time"
+
+	"github.com/microsoft/durabletask-go/backend/azure/dtmb/internal/backend/v1"
+	"google.golang.org/grpc/metadata"
+)
+
+var UserAgent = "dev/1"
+
+func ConnectWorker(
+	ctx context.Context,
+	testID string,
+	taskHub string,
+	worker backend.TaskHubWorkerClient,
+	orchestratorFns OrchestratorFnList,
+	activityFns ActivityFnList,
+	serverMsgChan chan<- *backend.ConnectWorkerServerMessage,
+	clientMsgChan <-chan *backend.ConnectWorkerClientMessage,
+	readyCb func(),
+	debug bool,
+) error {
+	defer close(serverMsgChan)
+
+	// Establish the ConnectWorker stream
+	streamCtx := metadata.AppendToOutgoingContext(ctx,
+		"taskhub", taskHub,
```

Comment:
Nit: use a constant for key

Reply:
Yeah, like I said this PR is nowhere near final - my TODO is to make a lot of constants configurable where the backend is initiated.